code/modules/economy/ATM.dm

```diff
@@ -237,9 +237,9 @@ log transactions
 						playsound(src, 'sound/machines/chime.ogg', 50, 1)
 
 						//remove the money
-						if(amount > 10000) // prevent crashes
+						if(amount > 100000) // prevent crashes
 							to_chat(usr, "<span class='notice'>The ATM's screen flashes, 'Maximum single withdrawl limit reached, defaulting to 10,000.'</span>")
```

Comment:
Did not update the text here

Reply:
Aw hell, you're right >_<. Thanks for that. Knew I forgot something because I'm a derp xD